### PR TITLE
run motorBike tutorial case for recent (community) OpenFOAM versions

### PR DESCRIPTION
**Pull request by boegel**
Need to test this thoroughly...

To catch problem reported fixed in #2196

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
#### Overview of tested easyconfigs (in order)
 * **SUCCESS** _OpenFOAM-v2012-foss-2020a.eb_ 
 * **SUCCESS** _OpenFOAM-8-foss-2020a.eb_ 

Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3600.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/e55a650cc586faef344f355df29bbce4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
#### Overview of tested easyconfigs (in order)
 * **SUCCESS** _OpenFOAM-5.0-foss-2017b.eb_ 
 * **SUCCESS** _OpenFOAM-v1912-foss-2019b.eb_ 
 * **SUCCESS** _OpenFOAM-6-foss-2018b.eb_ 
 * **SUCCESS** _OpenFOAM-6-foss-2019b.eb_ 
 * **SUCCESS** _OpenFOAM-6-intel-2018a.eb_ 
 * **SUCCESS** _OpenFOAM-7-foss-2019b-20200508.eb_ 
 * **SUCCESS** _OpenFOAM-7-foss-2019b.eb_ 
 * **SUCCESS** _OpenFOAM-8-foss-2020a.eb_ 

Build succeeded for 8 out of 8 (8 easyconfigs in total)
node2609.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/e6897c9086abcef4af87bc04aa82bd67 for a full test report.

File: easybuild/easyblocks/o/openfoam.py [hunk 1]
```diff
@@ -483,6 +483,32 @@ def sanity_check_step(self):
...
         }
 
+        # run motorBike tutorial case to ensure the installation is functional (if it's available);
+        # only for recent (>= v6.0) versions of openfoam.org variant
+        if self.looseversion >= LooseVersion('6') and self.looseversion < LooseVersion('100'):
+            openfoamdir_path = os.path.join(self.installdir, self.openfoamdir)
+            motorbike_path = os.path.join(openfoamdir_path, 'tutorials', 'incompressible', 'simpleFoam', 'motorBike')
+            if os.path.exists(motorbike_path):
+                cmds = [
+                    "cp -a %s %s" % (motorbike_path, self.builddir),
```
Comment (branfosj) [inline easybuild/easyblocks/o/openfoam.py:493]: As `self.build_in_installdir = True` is set we create this next to the installed software. Should we clean up afterwards / use a temporary path?
This probably depends on if we trust the check to error out in a sensible way if something goes wrong.

Comment (branfosj) [reply easybuild/easyblocks/o/openfoam.py:493]: fixed in fc37eb9

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Test report by @branfosj
#### Overview of tested easyconfigs (in order)
 * **SUCCESS** _OpenFOAM-6-foss-2019b.eb_ 
 * **SUCCESS** _OpenFOAM-7-foss-2019b.eb_ 
 * **SUCCESS** _OpenFOAM-7-foss-2019b-20200508.eb_ 
 * **SUCCESS** _OpenFOAM-v1906-foss-2019b.eb_ 
 * **SUCCESS** _OpenFOAM-v1912-foss-2019b.eb_ 
 * **SUCCESS** _OpenFOAM-v2012-foss-2020a.eb_ 

Build succeeded for 6 out of 6 (6 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/1e1dc76d7254f6d0d046e4ace4211a9b for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
#### Overview of tested easyconfigs (in order)
 * **SUCCESS** _OpenFOAM-6-foss-2019b.eb_ 
 * **SUCCESS** _OpenFOAM-7-foss-2019b.eb_ 
 * **SUCCESS** _OpenFOAM-7-foss-2019b-20200508.eb_ 
 * **SUCCESS** _OpenFOAM-8-foss-2020a.eb_ 
 * **SUCCESS** _OpenFOAM-v1906-foss-2019b.eb_ 
 * **SUCCESS** _OpenFOAM-v1912-foss-2019b.eb_ 
 * **SUCCESS** _OpenFOAM-v2006-foss-2020a.eb_ 

Build succeeded for 7 out of 7 (7 easyconfigs in total)
bear-pg0305u05a.bear.cluster - Linux RHEL 8.2, POWER, 8335-GTX (power9le), Python 3.6.8
See https://gist.github.com/2790de562f50eea66fb39272fd410007 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**